drivers/ksfilter/ks/connectivity.c

```diff
@@ -303,12 +307,15 @@ KspReadMediaCategory(
     /* now read the info */
     Status = ZwQueryValueKey(hKey, &Name, KeyValuePartialInformation, (PVOID)KeyInfo, Size, &Size);
 
+    DPRINT("ZwQueryValueKey() status 0x%08lx %wZ\n", Status, &Name);
+
```

Comment:
I see no point in logging the status code here as most of the time it is `STATUS_SUCCESS` anyway.